### PR TITLE
Geek-onacci Number in C++

### PR DESCRIPTION
**Pull request by U-c0de**
Added code in C++  [#1764]

Please Check-  @rish-singhal @Rohan27s @rudrakshi99

### CODE REVIEW

File: Code/C++/readme.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
  # Codes in the C++ language
 
-Unique Permutations of String ----> [C++](/Code/C++/unique_permutaions_of_string.cpp)
\ No newline at end of file
+Unique Permutations of String ----> [C++](/Code/C++/unique_permutaions_of_string.cpp)
+Geek-onacci Number----> [C++](/Code/C++/Geek-onacciNumber.cpp)
```
Comment (Rohan27s) [inline Code/C++/readme.md:4]: make these changes in Algo-Tree/Algorithm/Number_theory/ and not in Code/C++/readme.md

**Review by Rohan27s — CHANGES_REQUESTED** [review]

**Comment by U-c0de** [comment]
@Rohan27s  
Updated Do check

File: Code/C++/readme.md (hunk 1 above)
Comment (U-c0de) [reply Code/C++/readme.md:4]: @Rohan27s  ReadMe updated please check

File: Algorithm/Number_theory/readme.md [hunk 2]
```diff
@@ -37,13 +33,12 @@
...
 
-## Questions related to Number Theory
-- Counting number of digits in factorial using Kamenetsky's Formula ----> [Java](Code/Java/Kamenetsky_Formula.java)
 - GCD of two number using Euclidean Algorithm ----> [Java](/Code/Java/GCD.java)
-- All Unique Prime Factors ----> [Java](/Code/Java/Uniquefactor.java)
-- Check Prime Number ----> [Java](/Code/Java/check_prime.java)
-- Fibonacci Series ----> [Java](/Code/Java/Fibonacci_Series.java)
-- Determining if a number is a prime number or not using Trial Division Algorithm ---> [C++](/Code/C++/prime_number.cpp)
-- Factorial of a number without recursion ---->[Python](/Code/Python/factorial_without_recursion.py)
-- Check Trimorphic number ---->[C++](/Code/C++/Trimorphic_number.cpp)
```
Comment (Rohan27s) [inline Algorithm/Number_theory/readme.md:49]: don't make any othet changes on this readme file.... don't delete already existing text... just add your part at last

File: Code/C++/Geek-onacciNumber.cpp [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    return 0;
+}
+
+/*
+Time complexity : O(log n)
+Space complexity : O(1)
+*/
+
+/*
+Test Case :
```
Comment (Rohan27s) [inline Code/C++/Geek-onacciNumber.cpp:44]: add 2 test cases

File: Code/C++/readme.md [hunk 4]
```diff
@@ -1,3 +1,4 @@
  # Codes in the C++ language
 
-Unique Permutations of String ----> [C++](/Code/C++/unique_permutaions_of_string.cpp)
\ No newline at end of file
+Unique Permutations of String ----> [C++](/Code/C++/unique_permutaions_of_string.cpp)
```
Comment (Rohan27s) [inline Code/C++/readme.md:3]: don't commit in this readme

**Review by Rohan27s — CHANGES_REQUESTED** [review]

**Comment by Rohan27s** [comment]
Resolve conflicts

**Comment by U-c0de** [comment]
@Rohan27s  conflicts are resolved 
I had made a new pull request #1793 Please check it

**Review by Rohan27s — COMMENTED** [review]
Duplicate PR

**Pull request closed without merging**